llms/mlx_lm/generate.py

```diff
@@ -14,6 +15,10 @@
 DEFAULT_SEED = 0
 
 
+def str2bool(string):
+    return string.lower() not in ["False", "F"]
```

Comment:
Looks like that will never return `True` since you lower case the string before the check:
```suggestion
    return string not in ["False", "F"]
```

Reply:
@awni Thanks for pointing out that typo.  I wanted the option to work for 'T', 'True', 'F', and 'False' without regard to case, so I changed it to:

```python
def str2bool(string):
    return string.lower() not in ["false", "f"]
```